app/forms/field_handlers/field_handler.py

```diff
@@ -18,20 +18,17 @@ class FieldHandler(ABC):
     def __init__(
         self,
         answer_schema: dict,
+        value_source_resolver: ValueSourceResolver,
         error_messages: dict = None,
-        answer_store: AnswerStore = None,
-        metadata: dict = None,
-        location: Location = None,
         disable_validation: bool = False,
         question_title: str = None,
     ):
-        self.answer_schema = answer_schema
+        self.value_source_resolver = value_source_resolver
         self.error_messages = error_messages or {}
-        self.answer_store = answer_store or AnswerStore()
-        self.metadata = metadata or {}
-        self.location = location
+        self.answer_schema = answer_schema
```

Comment:
Minor, but these should match the init param order.
```python
        self.answer_schema = answer_schema
        self.value_source_resolver = value_source_resolver
        self.error_messages = error_messages or {}
```

Reply:
Resolved in https://github.com/ONSdigital/eq-questionnaire-runner/commit/fdc18d4747444751dbb2bea1da8180d1eea9d06e